src/streaming/controllers/BufferController.js

```diff
@@ -281,6 +281,13 @@ function BufferController(config) {
         if (appendedBytesInfo.segmentType === HTTPRequest.MEDIA_SEGMENT_TYPE) {
             showBufferRanges(ranges);
             onPlaybackProgression();
+
+            // If seeking, seek video model to range start in case appended segment starts beyond seek target
+            if (!isNaN(seekTarget) &&
+                (playbackController.getTime() === 0 || playbackController.getTime() < ranges.start(0))) {
+                playbackController.seek(ranges.start(0), true, true);
```

Comment:
Do we need to consider seekTarget here? We check if seekTarget is defined but do not use it. seekTarget < ranges.start(0) ?

Reply:
It replaces somehow the process in PlaybackController::onBytesAppended()